tests/std/tests/P1147R1_printing_volotile_pointers/test.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstddef>
+#include <sstream>
+
+using namespace std;
+
+std::string getExpected(int* ptr) {
+    ostringstream out;
+    out << ptr;
+    return out.str();
+}
+
+std::string getActual(volatile int* ptr) {
+    ostringstream out;
+    out << ptr;
+    return out.str();
+}
+
+void test(size_t value) {
+    int* p0          = reinterpret_cast<int*>(value);

```

Comment:
Why reinterpret a pointer-to-`size_t` - which admittedly is fine on our supported platforms - instead of simply taking the address of an `int`?

Reply:
This isn't converting a pointer-to-`size_t`, it's converting a `size_t` to a pointer-to-`int` which, again, relies on platform-specific guarantees. I'd still be more comfortable if we used a valid pointer value for this test.